app/src/components/experiment/AnnotationLabel.tsx

```diff
@@ -39,121 +22,88 @@ const textCSS = css`
     text-overflow: ellipsis;
   }
 `;
+
+const getAnnotationDisplayValue = (
+  annotation: Annotation,
+  display: AnnotationDisplay
+) => {
+  switch (display) {
+    case "label":
+      return (
+        annotation.label ||
+        (typeof annotation.score == "number" &&
+          formatFloat(annotation.score)) ||
+        "n/a"
+      );
+    case "score":
+      return (
+        (typeof annotation.score == "number" &&
+          formatFloat(annotation.score)) ||
+        annotation.label ||
+        "n/a"
+      );
+    default:
+      assertUnreachable(display);
+  }
+};
+
 export function AnnotationLabel({
   annotation,
   onClick,
+  annotationDisplay = "score",
 }: {
   annotation: Annotation;
   onClick?: () => void;
+  annotationDisplay?: AnnotationDisplay;
 }) {
   const clickable = typeof onClick == "function";
-  const labelValue =
-    (typeof annotation.score == "number" && formatFloat(annotation.score)) ||
-    annotation.label ||
-    "n/a";
+  const labelValue = getAnnotationDisplayValue(annotation, annotationDisplay);
 
   return (
-    <TooltipTrigger delay={500} offset={3}>
-      <TriggerWrap>
+    <div
+      role="button"
+      css={css`
+        cursor: ${clickable ? "pointer" : "default"};
```

Comment:
small change here since some of these are no longer clickable

Reply:
I would move this into it's own pre-compiled CSS block so that the css rule is not getting re-added over and over.